login.go

```diff
@@ -19,10 +21,25 @@ type LoginCmd struct {
 	cacheDir    string
 	githubToken string
 	role        string
+	Force       bool `arg:"--force, -f" help:"grab fresh tokens, ignoring the cache"`
 }
 
 const githubApi = "api.github.com"
 
+func (l *LoginCmd) isNotExpired(tokenPath string) bool {
+	if l.Force {
+		return false
+	}
+
+	fileStat, err := os.Stat(tokenPath)
+
+	if err != nil {
+		logger.Fatal(err)
```

Comment:
So this should probably be `return false`, because if you have no files in your cache yet, it will exit immediately otherwise.

Reply:
ah yeah! Good catch, I could have swore I tested that, but I guess that's what real tests are for :smile: 
Should be fixed

Reply:
`logger.Fatal` will exit immediately, so this will still not work until you remove that logger line.

Reply:
:facepalm: